src/components/profile/profile.component.js

```diff
@@ -103,14 +87,8 @@ Profile.propTypes = {
   email: PropTypes.string.isRequired,
   /** Define initials to display if there is no Gravatar image. */
   initials: PropTypes.string,
-  /** [Legacy] Enable a larger theme for the name. */
-  large: PropTypes.bool,
```

Comment:
that should be done as a separate commit with it's own breaking change

Reply:
It's not really a breaking change as `large` doesn't do anything